packages/flame/lib/src/cache/images.dart

```diff
@@ -45,6 +45,19 @@ class Images {
     _assets[name] = _ImageAsset.fromImage(image);
   }
 
+  /// Returns the image [name] from the cache. If no image with [name] exists
+  /// uses [imageGenerator] to generate and then add the resulting image into
+  /// the cache.
+  /// The cache will assume the ownership of the resulting image, and will
+  /// properly dispose of it at the end.
```

Comment:
Nit: This documentation requires formatting to comply with effective Dart.

I'm using a phone so I cant give a proper formatted example.

However, the first line/s should be as brief and separated by a new line. 

In addition we should avoid being redundant with what the function signature is already telling us. For example, "Returns an image" is already something we now from the signature.

Reply:
Updated